circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/standard_circuit_builder.cpp

```diff
@@ -240,6 +240,237 @@ template <typename FF> void StandardCircuitBuilder_<FF>::create_poly_gate(const
     ++this->num_gates;
 }
 
+/**
+ * @brief Create a gate where we validate an elliptic curve point addition
+ *        (x1, y1) + (x2, y2) = (x3, y3)
+ *        N.B. uses incomplete addition formula. Use with caution
```

Comment:
can you specify what the output will be if one use this function and there is a collision?